_sql/hotelify.sql

```diff
@@ -434,14 +540,20 @@ INSERT INTO `room_type` (`id`, `type_name`, `occupancy`, `description`, `price`,
 (38, 'Standard King Room', 5, 'this room is for kanglong', 300, 3),
 (39, 'Deluxe President Suite', 5, 'For Hotelify developers only', 1000, 1),
 (40, 'Deluxe King Room', 2, 'Deluxe King', 800, 2),
-(41, 'Deluxe Queen Room', 2, NULL, 700, 2),
-(42, 'Standard Queen Room', 2, NULL, 150, 5),
-(43, 'Standard Double Room', 4, NULL, 300, 2),
+(41, 'Deluxe Queen Room', 2, 'Deluxe Queen', 700, 2),
+(42, 'Standard Queen Room', 2, 'Standard Queen', 150, 5),
+(43, 'Standard Double Room', 4, 'Standard double', 300, 2),
 (44, 'Deluxe Queen Room', 2, 'Great views', 150, 2),
 (45, 'Deluxe Queen Room', 2, 'Great views', 190, 1),
-(46, 'Deluxe King Room', 2, NULL, 400, 2),
-(47, 'Regular Double Bed Room', 2, NULL, 100, 1),
-(48, 'Standard Queen Room', 2, NULL, 400, 5);
+(48, 'Standard Queen Room', 2, 'Standard Queen', 400, 5),
+(49, 'Deluxe King Room', 2, 'Room with great city view', 700, 1),
+(50, 'Regular Double Bed Room', 2, 'Extra beds available', 400, 0),
+(51, 'City View King Room', 2, 'Luxury room with great city view of Downtown Seattle', 500, 5),
```

Comment:
damn, really good efforts here! Do we have any views that can reflect these room type descriptions? @suiyoubi

Reply:
we do